includes/discovery/ports.inc.php

```diff
@@ -169,6 +174,11 @@
 
 echo "\n";
 
+//Nokia context ihub
```

Comment:
What is the purpose of this bit of code?

Reply:
It was to set the contact back to previous, but I think it's obsolete and can be removed